selfdrive/car/gm/gmcan.py

```diff
@@ -84,14 +84,21 @@ def create_friction_brake_command(packer, bus, apply_brake, idx, near_stop, at_f
 
   return packer.make_can_msg("EBCMFrictionBrakeCmd", bus, values)
 
-def create_acc_dashboard_command(bus, acc_engaged, target_speed_ms, lead_car_in_sight):
-  engaged = 0x90 if acc_engaged else 0
-  lead_car = 0x10 if lead_car_in_sight else 0
-  target_speed = int(target_speed_ms * 208) & 0xfff
-  speed_high = target_speed >> 8
-  speed_low = target_speed & 0xff
-  dat = [0x01, 0x00, engaged | speed_high, speed_low, 0x01, lead_car]
-  return [0x370, 0, "".join(map(chr, dat)), bus]
+def create_acc_dashboard_command(packer, bus, acc_engaged, target_speed_kph, lead_car_in_sight):
+  # Not a bit shift, dash can round up based on low 4 bits.
+  target_speed = int(target_speed_kph * 16) & 0xfff
```

Comment:
Ah, nice catch on the speed conversion.
should we change the dbc unit to 0.0625 and remove the factor 16?